content/docs/1.2.4/deploy/important-notes/index.md

```diff
@@ -8,4 +8,5 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
 
 ## Notes
 1. Please ensure your Kubernetes cluster is at least v1.18 before upgrading to Longhorn v{{< current-version >}} because the supported Kubernetes version has been updated (>= v1.18) in v{{< current-version >}}.
-1. After the upgrade, the recurring job settings of volumes will be migrated to new recurring job resources, and the `RecurringJobs` field in the volume spec will be deprecated. [[doc](https://longhorn.io/docs/{{< current-version >}}/deploy/upgrade/#4-automatically-migrate-recurring-jobs)]
+2. After the upgrade, the recurring job settings of volumes will be migrated to new recurring job resources, and the `RecurringJobs` field in the volume spec will be deprecated. [[doc](https://longhorn.io/docs/{{< current-version >}}/deploy/upgrade/#4-automatically-migrate-recurring-jobs)]
+3. When upgrading from a Longhorn version >= 1.2.0 and <= v1.2.3 to v{{< current-version >}}, if your cluster has many backups, you may expect to have a long upgrade time (with 22000 backups, it could take a few hours). Helm upgrade may timeout and you may need to re-run the upgrade command. This is [a known issue](https://github.com/longhorn/longhorn/issues/3890)
```

Comment:
```suggestion
3. When upgrading from a Longhorn version >= 1.2.0 and <= v1.2.3 to v{{< current-version >}}, if your cluster has many backups, you may expect to have a long upgrade time (with 22000 backups, it could take a few hours). Helm upgrade may timeout and you may need to re-run the upgrade command or set a longer timeout. This is [a known issue](https://github.com/longhorn/longhorn/issues/3890)
```